wsf_scraping/spiders/who_iris_spider.py

```diff
@@ -26,8 +26,9 @@ class WhoIrisSpider(scrapy.Spider):
         'JOBDIR': 'crawls/who_iris'
     }
 
-    def __init__(self, years_list='', feed_config=''):
+    def __init__(self, *args, **kwargs):
         settings = get_project_settings()
+        years_list = kwargs.get('years_list', False)
```

Comment:
We can just run the web scrapper with default parameters, the it won't have any args passed, and so, no kwargs['years_list']. I set it to false in that case so that it uses the settings['WHO_IRIS_YEARS'] in the next if statement